Sources/ButtonBarViewCell.swift

```diff
@@ -26,19 +26,24 @@ import Foundation
 
 open class ButtonBarViewCell: UICollectionViewCell {
     
-    @IBOutlet open var imageView: UIImageView!
-    @IBOutlet open lazy var label: UILabel! = { [unowned self] in
-        let label = UILabel(frame: self.contentView.bounds)
-        label.autoresizingMask = [.flexibleWidth, .flexibleHeight]
-        label.textAlignment = .center
-        label.font = UIFont.boldSystemFont(ofSize: 14.0)
-        return label
-    }()
+    @IBOutlet weak open var imageView: UIImageView!
+    @IBOutlet weak open var label: UILabel!
+    
+    open override func awakeFromNib() {
+        super.awakeFromNib()
+        
+        let labelAux = label ?? UILabel()
+        labelAux.frame = self.contentView.bounds
+        labelAux.autoresizingMask = [.flexibleWidth, .flexibleHeight]
+        labelAux.textAlignment = .center
+        labelAux.font = UIFont.boldSystemFont(ofSize: 14.0)
```

Comment:
You are right, I consider not thoughtful. Since the 'lazy body' will not be executed, why not remove it? like imageView